Makefile

```diff
@@ -58,15 +58,16 @@ docs:
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 
-RAIDENVERSION ?= 'master'
+RAIDENVERSION?=master
+GETH_URL_LINUX?='https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.7.3-4bb3c89d.tar.gz'
+SOLC_URL_LINUX?='https://github.com/ethereum/solidity/releases/download/v0.4.18/solc-static-linux'
+
 bundle:
-	# pass RAIDENVERSION=<git version tag> to build a specific version
-	docker build -t raidenbundler --build-arg RAIDENVERSION=$(RAIDENVERSION) -f docker/build.Dockerfile docker
-	-(docker rm bundler)
-	docker run --name bundler --privileged -e ARCH=x86_64 -e APP=raiden -e LOWERAPP=raiden --workdir / --entrypoint /bin/bash raidenbundler -c 'source functions.sh && generate_appimage'
-	mkdir -p dist
-	docker cp bundler:/out/raiden-.glibcPRIVATE-x86_64.AppImage dist/raiden--x86_64.AppImage
-	docker rm bundler
+	docker build -t pyinstallerbuilder --build-arg GETH_URL_LINUX=$(GETH_URL_LINUX) --build-arg SOLC_URL_LINUX=$(SOLC_URL_LINUX) --build-arg RAIDENVERSION=$(RAIDENVERSION) -f docker/build.Dockerfile docker
+	-(docker rm builder)
+	docker create --name builder pyinstallerbuilder
+	docker cp builder:/raiden/raiden-$(RAIDENVERSION).tar.gz dist/raiden-$(RAIDENVERSION).tar.gz
```

Comment:
It would be nice to use a consistent naming scheme for the binary packages. For example: `raiden-<version>-<platform>[-<optional>]` where `optional` could be `amd64` or `armhf` (if we ever decide to build an arm package for example)

Reply:
I added `-linux ` to the filenames, we can add the `<optional>` stuff later.